handlers/dashboard/publications.go

```diff
@@ -248,19 +227,19 @@ func (h *Handler) RefreshUPublications(w http.ResponseWriter, r *http.Request, c
 	facultyCols = append(facultyCols, "UGent", "-")
 	ptypes := append([]string{"all"}, vocabularies.Map["publication_types"]...)
 
-	locptypes := localize.VocabularyTerms(ctx.Loc, "publication_types")
+	locptypes := localize.VocabularyTerms(c.Loc, "publication_types")
 	locptypes["all"] = "All"
 
 	bindPublications := BindPublications{}
 	if err := bind.Request(r, &bindPublications); err != nil {
-		h.Logger.Warnw("publication dashboard could not bind request arguments", "errors", err, "request", r, "user", ctx.User.ID)
+		c.Log.Warnw("publication dashboard could not bind request arguments", "errors", err, "request", r, "user", c.User.ID)
 		render.BadRequest(w, r, err)
```

Comment:
use c.HandleError with httperror.BadRequest

Reply:
should be fixed